internal/services/cognitive/registration.go

```diff
@@ -56,5 +56,6 @@ func (r Registration) Resources() []sdk.Resource {
 	return []sdk.Resource{
 		AzureAIServicesResource{},
 		CognitiveDeploymentResource{},
+		CognitiveAccountRaiPolicyResource{},
```

Comment:
Alphabetical ordering, please move this to line 58

Reply:
Will do